.gitignore

```diff
@@ -9,3 +9,16 @@ erl_crash.dump
 
 # We have example files for docker-compose.yml so let's ignore it
 docker-compose.yml
+
+/node_modules
+
+# Ignore in umbrella apps node modules
+apps/*/node_modules/
+
+# Ignore generated files of in umbrella apps
```

Comment:
what about if we place a .gitignore in each app ? ... in webapp we might want to ignore all in **priv/static** but maybe in another app we have some different criteria.